cartography/intel/aws/apigateway.py

```diff
@@ -0,0 +1,336 @@
+import json
+import logging
+
+from botocore.exceptions import ClientError
+from policyuniverse.policy import Policy
+
+from cartography.util import aws_handle_regions
+from cartography.util import run_cleanup_job
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+
+
+@timeit
+@aws_handle_regions
+def get_apigateway_rest_apis(boto3_session, region):
+    client = boto3_session.client('apigateway', region_name=region)
+    paginator = client.get_paginator('get_rest_apis')
+    apis = []
+    for page in paginator.paginate():
+        apis.extend(page['items'])
+    return apis
```

Comment:
Do you think we need to do as_dict() similar to what we did for KMS?

Reply:
We didn't use as_dict() in KMS. That was the Azure codes